server/src/serializer/ShoeSerializer.js

```diff
@@ -5,14 +5,27 @@ class ShoeSerializer {
     static async getDetail(shoe) {
         const allowedAttributes = ["id", "name", "category", "color", "description", "url"]
         let serializedShoe = {}
+
         for (const attribute of allowedAttributes) {
             serializedShoe[attribute] = shoe[attribute]
         }
+
         const relatedReviews = await shoe.$relatedQuery("reviews")
         const serializedReviews = await Promise.all(
           relatedReviews.map(async(review) => await ReviewSerializer.getSummary(review))
         )
-        serializedShoe.reviews = serializedReviews
+
+        const sortedSerializedReviews = serializedReviews.sort((a,b) => {
+          if ( a.votes < b.votes ){
+            return 1
+          }
+          if ( a.votes > b.votes ){
+            return -1
+          }
+          return 0
+        }) 
```

Comment:
I'd recommend this review sorting to be a method in the review serializer so that we can also sort reviews in other places of the app, not just in this function/method